packages/amplify_datastore/example/ios/unit_tests/DataStoreHubEventStreamHandlerTests.swift

```diff
@@ -25,11 +25,12 @@ let testHubSchema: ModelSchema = ModelSchema.init(name: "Post")
 class DataStoreHubEventStreamHandlerTests: XCTestCase {
 
     var pluginUnderTest: SwiftAmplifyDataStorePlugin = SwiftAmplifyDataStorePlugin()
-    var flutterModelSchemaRegistration: FlutterModels = FlutterModels()
+    var modelSchemaRegistry = FlutterSchemaRegistry()
+    var customTypeSchemaRegistry = FlutterSchemaRegistry()
 
     override func setUpWithError() throws {
-        flutterModelSchemaRegistration.addModelSchema(modelName: "Post", modelSchema: testSchema)
-        flutterModelSchemaRegistration.registerModels(registry: ModelRegistry.self)
+        modelSchemaRegistry.addModelSchema(modelName: "Post", modelSchema: testSchema)
+        modelSchemaRegistry.registerModels(registry: ModelRegistry.self)
```

Comment:
Can we test custom types here?

Reply:
This `Post` added here will be used for `HubPayload.EventName.DataStore.outboxMutationEnqueued` and `HubPayload.EventName.DataStore.outboxMutationProcessed` within `DataStoreHubEventStreamHandler`, which invoke `FlutterSerializedModel#toMap` method. The mocked model schema is used here.

`FlutterSerializedModel#toMap` has been tested in `AmplifySerializedModelUnitTests`. Hence, I don't think it's necessary here to add custom type to the mocked model.